rmc/src/lib.rs

```diff
@@ -184,21 +184,21 @@ impl<T: Send + Sync + Clone> TaskScheduler<T> for DoublingDelayScheduler<T> {
 ///
 /// We refer to the documentation https://cardinal-cryptography.github.io/AlephBFT/reliable_broadcast.html
 /// for a high-level description of this protocol and how it is used for fork alerts.
-pub struct ReliableMulticast<'a, H: Signable + Hash, MK: MultiKeychain> {
+pub struct ReliableMulticast<H: Signable + Hash, MK: MultiKeychain> {
     hash_states: HashMap<H, PartiallyMultisigned<H, MK>>,
     network_rx: UnboundedReceiver<Message<H, MK::Signature, MK::PartialMultisignature>>,
     network_tx: UnboundedSender<Message<H, MK::Signature, MK::PartialMultisignature>>,
-    keychain: &'a MK,
+    keychain: MK,
     scheduler: Box<dyn TaskScheduler<Task<H, MK>>>,
     multisigned_hashes_tx: UnboundedSender<Multisigned<H, MK>>,
     multisigned_hashes_rx: UnboundedReceiver<Multisigned<H, MK>>,
 }
 
-impl<'a, H: Signable + Hash + Eq + Clone + Debug, MK: MultiKeychain> ReliableMulticast<'a, H, MK> {
+impl<'a, H: Signable + Hash + Eq + Clone + Debug, MK: MultiKeychain> ReliableMulticast<H, MK> {
```

Comment:
Why is the `'a` still here?

And in general thanks for removing it everywhere, it was a bit of a leftover from a botched cleanup anyway. :P